tests/test_scripts.py

```diff
@@ -514,6 +514,21 @@ def test_normalize_by_median():
     assert len(seqs) == 1, seqs
     assert seqs[0].startswith('GGTTGACGGGGCTCAGGGGG'), seqs
 
+    # Testing for the apppend option
```

Comment:
please make a separate test function instead -- more granular tests are generally better. So, 

* copy the test function twice
* rename both copies
* remove the outfile/append tests from the original
* have one copy do the append test
* have the other copy do the not-append test

thanks!